.github/workflows/e2e-test-docker-image-build.yml

```diff
@@ -30,9 +47,10 @@ jobs:
         id: build-image
         env:
           REGISTRY: ${{ steps.login-ecr-public.outputs.registry }}
-          REGISTRY_ALIAS: h6o3z5z9
-          REPOSITORY: aws-application-signals-test-framework-workflow-container
+          REGISTRY_ALIAS: v6t6x3y1
```

Comment:
Thought we were going to have a custom alias now?
https://docs.aws.amazon.com/AmazonECR/latest/public/public-registry-settings.html

Reply:
This isn't needed anymore. We retrieve the url programmatically